metamorph/src/main/java/org/metafacture/metamorph/maps/FileMap.java

```diff
@@ -85,13 +88,14 @@ public void setFile(final String file) {
                 }
                 final String[] parts = split.split(line);
                 if (parts.length == 2) {
-                    map.put(parts[0], parts[1]);
+                    modifiableMap.put(parts[0], parts[1]);
                 }
             }
         }
         catch (final IOException | UncheckedIOException e) {
             throw new MorphExecutionException("filemap: cannot read map file", e);
         }
+        unmodifiableMap = Collections.unmodifiableMap(modifiableMap);
```

Comment:
This prevents loading multiple files (e.g., via `setFiles()`).

Reply:
uh, oh, no this is not good